block-producer/src/server/mod.rs

```diff
@@ -53,17 +53,13 @@ pub async fn serve(config: BlockProducerConfig) -> Result<()> {
     let block_producer = api_server::ApiServer::new(api::BlockProducerApi::new(queue.clone()));
 
     tokio::spawn(async move {
-        queue
-            .run()
-            .instrument(info_span!(target: COMPONENT, "transaction_queue_start"))
-            .await
+        info!(target: COMPONENT, "Transaction queue started");
+        queue.run().await
     });
```

Comment:
Should these be in reverse order? For exmaple:
```Rust
tokio::spawn(async move {
    queue.run().await;
    info!(target: COMPONENT, "Transaction queue started");
});
```

Reply:
No, since `queue.run().await` starts infinite loop. But we could move this info into the `run` method.